src/components/Footer.js

```diff
@@ -1,8 +1,127 @@
-import React from "react";
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faLinkedin, faGithub } from '@fortawesome/free-brands-svg-icons';
 
 const Footer = () => {
+  const currentYear = () => {
+    const today = new Date();
+    const year = today.getFullYear();
+    return year;
```

Comment:
```suggestion
    return today.getFullYear();
```

Since we are returning in the next line, we can just return the expression immediately so we don't have to create a new variable